environment.yml

```diff
@@ -25,5 +25,9 @@ dependencies:
     - nanofilt>=2.3
     - nanoplot>=1.20
     - hifiasm>=0.19
-    - hifiasm_meta>=hamtv0.3
+    - hifiasm_meta=hamtv0.2
```

Comment:
`hifiasm_meta` tests running version `hamtv0.3` are failing due to the lack of memory and "complexity of the problem" on my 64 gb of RAM laptop. When reverting back down to `hamtv0.2`, these tests pass normally. As of now, pinning the version number to keep `hifiasm_meta` tests running without crashing.